dockers/docker-sonic-telemetry/telemetry.sh

```diff
@@ -7,7 +7,7 @@ TELEMETRY=`sonic-cfggen -d -v 'TELEMETRY.keys() | join(" ") if TELEMETRY'`
 
 TELEMETRY_ARGS=" -logtostderr"
 
-if [ ! -z $X509 ]; then
+if [ ! -z "$X509" ]; then
```

Comment:
Also suggest simplifying. `! -z` == `-n`, thus,

```suggestion
if [ -n "$X509" ]; then
```